.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
-{
-    "files.associations": {
-        "orxandroid.h": "c"
-    }
-}
\ No newline at end of file
+{
+  "files.associations": {
+    "orxandroid.h": "c"

```

Comment:
Should this one be case sensitive for Linux?

Reply:
This was added in https://github.com/orx/orx/commit/0220784a314441b8283b701622780f8ef1212cc6 and looks like the naming may have been an oversight. I'm not sure if/why this is needed but I'll update the case.

Reply:
This should be addressed in https://github.com/orx/orx/pull/104/commits/3b8d6d54c33fe477230ab62464e69f4a46fc1112 as well.